paper/paper/src/main/java/dev/triumphteam/gui/paper/container/type/HopperContainerType.java

```diff
@@ -0,0 +1,47 @@
+package dev.triumphteam.gui.paper.container.type;
+
+import dev.triumphteam.gui.container.type.GuiContainerType;
+import dev.triumphteam.gui.exception.TriumphGuiException;
+import dev.triumphteam.gui.slot.Slot;
+import net.kyori.adventure.text.Component;
+import org.bukkit.Bukkit;
+import org.bukkit.event.inventory.InventoryType;
+import org.bukkit.inventory.Inventory;
+import org.bukkit.inventory.InventoryHolder;
+import org.jetbrains.annotations.NotNull;
+
+public class HopperContainerType implements PaperContainerType {
+
+    private static final int LOWER_LIMIT = 0;
+    private static final int UPPER_LIMIT = 5;
+
+    public static @NotNull GuiContainerType of() {
```

Comment:
Not part of this file but since I can't reference others I'll ask here. You could also add it to the builder `Gui.hopper()` for a quicker way to use this.